rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc

```diff
@@ -7,16 +7,13 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 toc::[]
 {hcp-title} offers the following worker node instance types and sizes:
 
-[NOTE]
-====
-Currently, {hcp-title} supports a maximum of 500 worker nodes. Customers running {hcp-title} 4.14.x and 4.15.x clusters require a minimum z-stream version of 4.14.28 or 4.15.15 and greater to scale to 500 worker nodes. For earlier versions, the maximum is 90 worker nodes. 
-====
-
 include::modules/rosa-sdpolicy-am-aws-compute-types.adoc[leveloffset=+1]
 
 include::modules/rosa-sdpolicy-am-aws-compute-types-graviton.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
+For more information about the maximum number of supported worker nodes in a {hcp-title} cluster, see xref:../../rosa_planning/rosa-hcp-limits-scalability.adoc#rosa-hcp-limits-scalability[{hcp-title} limits and scalability].
```

Comment:
[Additional resources sections](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) should just be bullets with link text:
```suggestion
* xref:../../rosa_planning/rosa-hcp-limits-scalability.adoc#rosa-hcp-limits-scalability[{hcp-title} limits and scalability]
```